layers/+spacemacs/spacemacs-editing-visual/packages.el

```diff
@@ -263,11 +267,15 @@
         ("d" ahs-forward-definition)
         ("D" ahs-backward-definition)
         ("e" ahs-to-iedit :exit t)
+        ("h" highlight-symbol-at-point)
         ("n" spacemacs/quick-ahs-forward)
         ("N" spacemacs/quick-ahs-backward)
         ("p" spacemacs/quick-ahs-backward)
         ("R" ahs-back-to-start)
         ("r" ahs-change-range)
+        ("s" (if (configuration-layer/layer-usedp 'ivy)
```

Comment:
Usually you'd modify `spacemacs-symbol-highlight-transient-state-add-bindings` in both the ivy and helm layers, as done [here](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+completion/helm/packages.el#L49-L52) and [here](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+completion/ivy/packages.el#L38-L41) for the other bindings. I'm not sure why you chose not to do this but I recommend that you do.